tensorflow_datasets/core/download/checksums.py

```diff
@@ -19,17 +19,17 @@
 
 import os
 from typing import Any, Dict, Iterable, List, Optional
+import importlib_resources
 
 import dataclasses
 import tensorflow.compat.v2 as tf
 
 from tensorflow_datasets.core import utils
 
 
-_ROOT_DIR = os.path.normpath(os.path.join(os.path.dirname(__file__), '../..'))
-
 _CHECKSUM_DIRS = [
-    os.path.join(_ROOT_DIR, 'url_checksums'),
+  os.path.normpath(
+    importlib_resources.files('tensorflow_datasets') / 'url_checksums'),
```

Comment:
You could use: `utils.resources_path('tensorflow_datasets')` instead which is a wrapper of `importlib_resources.files`.

Reply:
done